### DISCUSSION TITLE
keep screen on

### DISCUSSION THREAD

**Discussion by perottosson** [quasarframework/quasar#13550] [CLI - Capacitor / Cordova modes]
When I am building an app with android studio I can put a flag in the manifest for keeping my screen on. KEEP_SCREEN_ON

Is it possible to add this while building cordova application?

**Comment by MilosPaunovic** [comment]
[https://www.npmjs.com/package/cordova-plugin-insomnia](https://www.npmjs.com/package/cordova-plugin-insomnia)

**Reply by perottosson** [reply]
Thanks. I installed it into src.cordova:
`$ cordova plugin add https://github.com/EddyVerbruggen/Insomnia-PhoneGap-Plugin.git
`
and after that:
`$ cordova prepare`

I guess it is working. Where do I put it for always being activated?  At the moment it falls a sleep. Somewhere in cordova.js?
